rever/main.xsh

```diff
@@ -159,5 +184,20 @@ def main(args=None):
         env_main(args=args)
 
 
+def setup(args=None):
```

Comment:
I think a better model is for each activity to have a setup or bootstrap method.  Then in the main file, yo just dispatch to the activities setup method

Reply:
Yes that is a good idea.